api/v1beta1/disk_failure.go

```diff
@@ -0,0 +1,46 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+package v1beta1
+
+import (
+	"fmt"
+	"strings"
+)
+
+// DiskFailureSpec represents a disk failure disruption
+type DiskFailureSpec struct {
+	Path string `json:"path"`
+}
+
+const MaxPathCharacters = 62
+
+// Validate validates args for the given disruption
+func (s *DiskFailureSpec) Validate() error {
+	if s.Path == "" {
+		return fmt.Errorf("the path of the disk failure disruption must not be empty")
+	}
+
+	path := strings.TrimSpace(s.Path)
+
+	if path == "" {
+		return fmt.Errorf("the path of the disk failure disruption must not be blank")
+	}
```

Comment:
```suggestion
	path := strings.TrimSpace(s.Path)

	if path == "" {
		return fmt.Errorf("the path of the disk failure disruption must not be empty")
	}
```

I think that's enough?

Reply:
Indeed, I can do it more simpler 👍